pkg/scaler/scaler.go

```diff
@@ -89,7 +89,8 @@ func NewController(clientset kubernetes.Interface,
 	})
 	c.secretsInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
 		AddFunc: func(new interface{}) {
-			klog.V(4).Infof("Secret Added\n")
+			newSecret := new.(*v1.Secret)
```

Comment:
if you are anyways casting the obj to `secret` then you can as well pass the same to `enqueueProbe` as well. This is handled in the refactoring that i am currently doing. So if you do not wish to include it as part of this PR then that would be fine.

Reply:
I was thinking we can keep it this way for now, so that we can get some detailed logs for testing the secret rotation change if required. Otherwise we would need to wait for refactored one for it.  wdyt?

Reply:
ok sure.